### ISSUE TITLE
Escaping code blocks within comment blocks

### ISSUE THREAD

**Issue by binarycow** [scriban/scriban#359]
Sometimes, I use a comment inside a code block as a header for the template.  Since I don't want the comment itself to be output to the template, so I need Scriban to interpret it - so I put it in the code block.

Example:

```
{{- ##
   This is a description of my template.
## -}}
```

Sometimes I want to put an example of a Scriban code block in the comment.

Example:

```
{{- ##

  If you want to use option A, paste the following line into your template:
     {{ "There are " + numberOfLions + " lions in the zoo." }}

## -}}
```

I would like the code block inside the comments to be escaped.

**Comment by xoofx** [comment]
> Currently, the Scriban renderer sees the `}}` inside the comment block to be the end of the comment block _and_ the outer code block.

Yes, the original behavior was to allow something like `{{ ## This is a comment }}` when you want to comment the rest of the function.

Any fix will have to preserve this behavior.

**Comment by InspiringCode** [comment]
It would be really helpful to have a way to include scriban examples with code blocks in comments. Is there currently ANY way to do this?
